src/main/g8/project/ProjectPlugin.scala

```diff
@@ -0,0 +1,97 @@
+import sbt._
+import sbt.Keys._
+import freestyle.rpc.idlgen.IdlGenPlugin.autoImport._
+import org.scalafmt.sbt.ScalafmtPlugin.autoImport._
+import sbt.{AutoPlugin, PluginTrigger}
+
+object ProjectPlugin extends AutoPlugin {
+
+  override def trigger: PluginTrigger = allRequirements
+
+  object autoImport {
+
+    lazy val V = new {
+      val fs2          = "0.10.1"
+      val log4cats     = "0.1.0"
+      val logbackClassic = "1.2.3"
```

Comment:
this is not properly formatted

Reply:
Done at https://github.com/frees-io/freestyle-rpc-seed.g8/pull/1/commits/b9cafb446430acecb89f7a43346e9f8bda88e0d3